financialaid/tasks_test.py

```diff
@@ -0,0 +1,78 @@
+"""
+Test for financialaid celery tasks
+"""
+from django.test import (
+    override_settings,
+    TestCase
+)
+from mock import patch
+
+from financialaid.models import CurrencyExchangeRate
+from financialaid.tasks import update_currency_exchange_rates
+
+
+@patch('financialaid.tasks.requests.get')
+class TasksTest(TestCase):
+    """
+    Tests for periodic task which which updates currency exchange rates from Open
+    Exchange Rates.
+    """
+
+    @classmethod
+    def setUpTestData(cls):
+        super(TasksTest, cls).setUpTestData()
+        CurrencyExchangeRate.objects.create(
+            currency_code="DEF",
+            exchange_rate=1.8
+        )
+        CurrencyExchangeRate.objects.create(
+            currency_code="MNO",
+            exchange_rate=2.1
+        )
+
+    def setUp(self):
+        super(TasksTest, self).setUp()
+        self.data = {
+            "extraneous information": "blah blah blah",
+            "rates": {
+                "DEF": "2",
+                "MNO": "1.7",
+                "PQR": "0.4"
+            }
+        }
+
+    @override_settings(CELERY_ALWAYS_EAGER=True)
+    def test_celery_task_works(self, mocked_request):  # pylint: disable=unused-argument
+        """
+        Assert task schedule using celery beat.
```

Comment:
I'm not sure this test is doing anything useful, and it doesn't seem to interact with celery beat at all. Can we delete it?